chords_start.sh

```diff
@@ -145,6 +145,10 @@ curl -s http://$influxdb_host:8086/query --data-urlencode "q=create user $influx
 # Make sure that the influxdb database exists.
 curl -s http://$influxdb_host:8086/query -u $influxdb_admin_user:$influxdb_admin_pw --data-urlencode "q=create database $influxdb_dbname"
 
+if [ "$RAILS_ENV" = "development" ]; then
```

Comment:
Why is the database chords_ts_test? Shouldn't it be chords_ts_development?

Reply:
The reasoning behind this is the assumption that when CHORDS is started with the RAILS_ENV=development that development work is being performed and that the test suite will also be run. 

The way I was running the test suite was with the command:
docker-compose run -e "RAILS_ENV=test" app bundle exec rspec

This command doesn't ever execute the chords_start.sh script, and hence the chords_ts_test database is never created, and rspec exits with a failure to connect.

Creating the test database when the containers are brought up with RAILS_ENV=development may not be the proper way to do this, but it works. I'm totally open to doing this another way if there are other suggestions. (But I'm not sure it's worth the time to go down this rabbit hole at this point)

There is likely a better was to address this issue, but I'm not sure it's worth the time to

Reply:
We should take a look at the various RAILS workflows (production, development and test). Perhaps the most consistent way to handle this is to make chords_control do the right thing for all of these. For now, this shouldn't hold up the merge of of this'll request.